dashboard/app/views/pd/application/teacher1819_application_mailer/principal_approval.html.haml

```diff
@@ -0,0 +1,42 @@
+%p
+  Dear
+  = @application.principal_title_and_last_name + ','
```

Comment:
What if they don't have a title? (not sure if that's possible with the form setup) Do we need a fallback to full name?

Reply:
Good point! In the original spec, title was required but we nixed that. @tanyaparker How should we address the email when there is no title? Full name? First name?

Reply:
Let's do full name.